caffe2/utils/threadpool/ThreadPool.cc

```diff
@@ -15,12 +15,18 @@ C10_DEFINE_int(caffe2_threadpool_android_cap, true, "");
 // Whether or not threadpool caps apply to iOS
 C10_DEFINE_int(caffe2_threadpool_ios_cap, true, "");
 
+C10_DEFINE_int(pthreadpool_size, 0, "Override the default thread pool size.");
+
 namespace caffe2 {
 
 size_t getDefaultNumThreads() {
   CAFFE_ENFORCE(cpuinfo_initialize(), "cpuinfo initialization failed");
   int numThreads = cpuinfo_get_processors_count();
 
+  if (FLAGS_pthreadpool_size) {
+    numThreads = FLAGS_pthreadpool_size;
```

Comment:
Since caffe2_threadpool_android_cap and caffe2_threadpool_ios_cap are set to true by default, overriding the thread count at this point is still going to let those caps override this setting.  Is that the intended behavior?

Reply:
Ah, no.  Good point.  I should move this down below that.  An explicit count should take priority.